storage/objects/main_test.go

```diff
@@ -224,6 +225,60 @@ func TestKMSObjects(t *testing.T) {
 	}
 }
 
+func TestV4SignedUrl(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	ctx := context.Background()
+	client, err := storage.NewClient(ctx)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
```suggestion
		r.Fatalf("storage.NewClient: %v", err)
```

`log.Fatal` will stop the test suite. `t.Fatal` fails the test (immediately).

Reply:
I'm following the existing code, should this be updated elsewhere as well?

Reply:
Yes, please, if you don't mind!